helm/ffc-demo-api-gateway/templates/deployment.yaml

```diff
@@ -2,15 +2,31 @@ apiVersion: extensions/v1beta1
 kind: Deployment
 metadata:
   name: {{ quote .Values.name }}
-  namespace: {{ quote .Values.namespace }}
+  labels:
+    app: {{ quote .Values.namespace }}
+    app.kubernetes.io/name: {{ quote .Values.name }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/version: {{ quote .Values.labels.version }}
+    app.kubernetes.io/component: {{ quote .Values.labels.component }}
+    app.kubernetes.io/part-of: {{ quote .Values.namespace }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    environment: {{ quote .Values.environment }}
```

Comment:
This block is repeated a few times in different places, would it be possible to extract it into its own file and use it across the templates by using the include function or rendering an external config file as described here? https://helm.sh/docs/howto/charts_tips_and_tricks/

Reply:
Label template used across this and other repos to reduce duplication.